sdk/tests/deqp/functional/gles3/fborender.html

```diff
@@ -13,7 +13,7 @@
 <body>
 <div id="description"></div>
 <div id="console"></div>
-<canvas id="canvas" width="200" height="100"> </canvas>
+<canvas id="canvas" width="200" height="128"> </canvas>
```

Comment:
Set the size the same as the one in C++ codes, which will make debugging easier.